src/app/features/issue/providers/gitlab/gitlab-issue/gitlab-issue.model.ts

```diff
@@ -41,7 +41,8 @@ export type GitlabIssue = Readonly<{
   comments: GitlabComment[];
   url: string;
   // NOTE: we use the issue number as id as well, as it there is not much to be done with the id with the api
-  id: number;
+  id: number | string;
```

Comment:
A comment would be helpful, to indicate why it now can be both string and number.